.github/workflows/release.yaml

```diff
@@ -1,36 +1,41 @@
-name: Release
```

Comment:
This is basically - delete release.yaml and replace with release-v2.yaml.  Since the new release process is working